.travis.yml

```diff
@@ -1,3 +1,6 @@
+plugins:
```

Comment:
This is definitely not the right file for this. Travis has nothing to do with Jekyll.